code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -87,7 +87,7 @@
 	/// Hacked state, see [/mob/living/simple_animal/drone/proc/update_drone_hack]
 	var/hacked = FALSE
 	/// If we have laws to minimize bothering others. Enables or disables drone laws enforcement components (use [/mob/living/simple_animal/drone/proc/set_shy] to set)
-	var/shy = FALSE //SKYRAT EDIT: Original: TRUE
+	var/shy = TRUE
```

Comment:
intended?

Reply:
yes, intended. I did it modular this time ahaha